code/game/machinery/adv_med.dm

```diff
@@ -483,11 +489,13 @@
 					infection = "Acute Infection:"
 				if(INFECTION_LEVEL_TWO + 200 to INFECTION_LEVEL_TWO + 300)
 					infection = "Acute Infection+:"
-				if(INFECTION_LEVEL_TWO + 300 to INFINITY)
+				if(INFECTION_LEVEL_TWO + 300 to INFECTION_LEVEL_TWO + 399)
 					infection = "Acute Infection++:"
+				if(INFECTION_LEVEL_TWO + 400 to INFINITY)
+					infection = "Septic:"
```

Comment:
It might be helpful to add a case between "Acute Infection++" and "Septic", since iirc mitocholide is needed once sepsis hits

Reply:
900 is when sepsis hits naturally in code, so for that I'd need to refactor code entirely or the print out itself would be inaccurate. Though mitocholide isn't needed for anything other than dead organs as far as I'm aware and remember with recent incidents.

Reply:
Are you sure? `necrotize` is called past INFECTION_LEVEL_THREE, which is at 1000 germs - and INFECTION_LEVEL_TWO is at 500 germs, so INFECTION_LEVEL_TWO+400 is 900 germs - which leaves a window of 100 germs where it would be displayed as "septic" despite not actually having necrotized yet.

Reply:
I'm quite sure, yes. I had to test germ levels fairly extensively, when I put germ level 900 on a limb it was septic, 900 being the first germ level it occurs at. 890-899 was acute affection++. It says this in the advanced scanner and did before I changed the code, the code that you're reading now only affects the print out not what the scanner tells you.

![GeVCuepXy7](https://user-images.githubusercontent.com/12178527/77238941-3fa76e00-6ba3-11ea-8756-5b8740e56eda.png)

This image is taken from my master without the scanner changes.

Reply:
Hmm, I guess it'd make sense in parity with the body scanner's nano interface. Sounds good!